modules/data.land/R/soil_process.R

```diff
@@ -30,6 +30,7 @@ soil_process <- function(settings, input, dbfiles, overwrite = FALSE,run.local=T
   # set up bety connection
   con <- PEcAn.DB::db.open(dbparms$bety)
   on.exit(PEcAn.DB::db.close(con), add = TRUE)
+  
   # get site info
   latlon <- PEcAn.DB::query.site(site$id, con = con)[c("lat", "lon")]
```

Comment:
Not following why this function only has whitespace change as it seems like the code chunk here is doing the same thing as what you were trying to support with your new function, and also would benifit from having a database-independent option.